inspirehep/modules/migrator/tasks.py

```diff
@@ -288,9 +289,23 @@ def create_record(record):
     return json
 
 
+def update_documents_and_figures_for_record(record):
+    for attachment in chain(record.get('documents', []), record.get('figures', [])):
+        file_url = attachment.get('url')
+        filename = attachment.get('key')
+        with file_resolving(file_url) as fp:
+            record.files[filename] = fp
```

Comment:
so you didn't implement what we discussed yesterday about mangling the `key` to include the `recid` in it?

Reply:
Yes I didn't include it. I will do it now!